roxie/ccd/ccdfile.cpp

```diff
@@ -190,7 +190,8 @@ class CRoxieLazyFileIO : implements ILazyFileIO, implements IDelayedFile, public
     {
         try
         {
-            if (current.get()!=&failure)
+            if (current.get()==&failure)
+                return;
             {
```

Comment:
Remove the curlies on this block (or reverse the sense of the test and move the current.set into the block)?